tests/functional/plugins/disabled_test_plugin.py

```diff
@@ -0,0 +1,17 @@
+class DisabledPluginModule:
+    def __init__(self):
+        self.invocations = 0
+
+    def register_event(self, client):
+        client.meta.events.register('before-call', self.increment_invocations)
+
+    def increment_invocations(self, **kwargs):
+        self.invocations += 1
+
+
+plugin_instance = DisabledPluginModule()
```

Comment:
Ideally we don't want to have this plugin at all, but due to limitations in the current design, we need an individual plugin per test (or a plugin per test in each plugin module). Moving to a setup where we can get a copy of the plugin per invocation will simplify testing/state issues.